cognite/neat/graph/loaders/rdf_to_dms.py

```diff
@@ -94,11 +94,10 @@ def load(self, stop_on_exception: bool = False) -> Iterable[InstanceApply | Erro
 
                     delta_time = datetime_utc_now() - start_time
                     delta_time = (delta_time.seconds * 1000000 + delta_time.microseconds) / 1000
-                    msg = (
+                    (
                         f"{class_name} {counter} instances processed, "
                         f"instance processing time: {delta_time/counter:.2f} "
                     )
```

Comment:
This does nothing now.
```suggestion
                    (
                        f"{class_name} {counter} instances processed, "
                        f"instance processing time: {delta_time/counter:.2f} "
                    )
```